File: docs/components_page/components/dropdown/style.jl
using DashBootstrapComponents

items = [
    dbc_dropdownmenuitem("Item 1"),
    dbc_dropdownmenuitem("Item 2"),
    dbc_dropdownmenuitem("Item 3"),
];

dropdowns = html_div(
    [
        dbc_dropdownmenu(items, label = "Primary", color = "primary", className = "m-1"),
        dbc_dropdownmenu(
            items,
            label = "Secondary",
            color = "secondary",
            className = "m-1",
        ),
        dbc_dropdownmenu(items, label = "Success", color = "success", className = "m-1"),
        dbc_dropdownmenu(items, label = "Warning", color = "warning", className = "m-1"),
        dbc_dropdownmenu(items, label = "Danger", color = "danger", className = "m-1"),
        dbc_dropdownmenu(items, label = "Info", color = "info", className = "m-1"),
    ],
    style = Dict("display" => "flex", "flexWrap" => "wrap"),
);

File: docs/components_page/components/dropdown/style.R
library(dashBootstrapComponents)

items_style <- list(
  dbcDropdownMenuItem("Item 1"),
  dbcDropdownMenuItem("Item 2"),
  dbcDropdownMenuItem("Item 3")
)

dropdowns <- div(
  list(
    dbcDropdownMenu(
      items_style,
      label = "Primary", color = "primary", className = "m-1"
    ),
    dbcDropdownMenu(
      items_style,
      label = "Secondary", color = "secondary", className = "m-1"
    ),
    dbcDropdownMenu(
      items_style,
      label = "Success", color = "success", className = "m-1"
    ),
    dbcDropdownMenu(
      items_style,
      label = "Warning", color = "warning", className = "m-1"
    ),
    dbcDropdownMenu(
      items_style,
      label = "Danger", color = "danger", className = "m-1"
    ),
    dbcDropdownMenu(items_style,
      label = "Info", color = "info",
      className = "m-1"
    )
  ),
  style = list(display = "flex", flexWrap = "wrap")
)
